### PR TITLE
Taxonomy edits 10 - category dedupes and translations

### PR DESCRIPTION
**Pull request by svensven**

### CODE REVIEW

File: taxonomies/categories.txt [hunk 1]
```diff
@@ -33565,54 +33479,63 @@ es:Quesucos de Liébana
 protected_name_file_number:en: PDO-ES-0090
 protected_name_type:en: pdo
 origins:en: en:Spain
+country:en:Spain
```
Comment (stephanegigandet) [inline taxonomies/categories.txt:33482]: In fact we need to remove all the country:en: entries, that was an experiment, but we use the countries field to indicate where the product is sold, not where it is from (we use the origins field for that). cc @teolemon

**Review by stephanegigandet — APPROVED** [review]
Looks good to me, thank you!

File: taxonomies/categories.txt (hunk 1 above)
Comment (svensven) [reply taxonomies/categories.txt:33482]: hmm, apparently there are 2217 "^country" entries. not one to edit by hand. :D

Comment (svensven) [reply taxonomies/categories.txt]: I've removed the "country:"s from categories that had an origins as well, so in theory, one could just search+replace the remaining country lines into origins ones?

Comment (svensven) [reply taxonomies/categories.txt]: also, is the space in the origins lines intentional?

Comment (Ban3) [reply taxonomies/categories.txt]: > also, is the space in the origins lines intentional?

I think it's there just to make the double `en:` clearer. Parser shouldn't care about that.

**Pull request merged**